### PR TITLE
Fix issue 70: isTest looks at superclass methods

### PR DESCRIPTION
**Pull request by aledsage**
- classes with superclasses with test methods now detected as tests

### CODE REVIEW

**Comment by missedone** [comment]
@aledsafe let's revisit this PR, do you still have issue #70 with latest vesion of testng eclipse plugin. if so, could you update this PR so that we can have a look then
thanks

**Comment by aledsage** [comment]
@missedone Thanks for looking at this! I confirmed this issue #70 was still an issue on master, and have re-applied my changes again master. PR is again ready for review.

File: testng-eclipse-plugin/src/main/org/testng/eclipse/util/TestSearchEngine.java [hunk 1]
```diff
@@ -204,49 +204,72 @@ public static boolean isTest(IJavaElement ije) {
...
     }
-    
+    catch(JavaModelException jme) {
+      TestNGPlugin.log(jme);
+    }
+    return false;
+  }
+  
+  private static boolean doIsTest(IType iType) {
+    ITestContent testContent = TypeParser.parseType(iType);
```
Comment (missedone) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/util/TestSearchEngine.java]: let's have a check if `iType` is abstract or not at the beginning of this method:
```java
if (Flags.isAbstract(iType.getFlags())) {
  return false;
}
```

**Comment by aledsage** [comment]
Thanks @missedone - added check for `isAbstarct`

**Pull request merged**